Update dashboard guide (#1233)
OttoWinter/esphomedocs: guides/getting_started_hassio.rst

===============================================

.. seo::
    :description: Getting Started guide for installing ESPHome Dashboard as a Home Assistant add-on and creating a basic configuration.
    :image: home-assistant.png

ESPHome is the perfect solution for creating custom firmware for
your ESP8266/ESP32 boards. In this guide we’ll go through how to setup a
basic "node" using the ESPHome Dashboard, installed as a Home Assistant add-on.

But first, here's a very quick introduction to how ESPHome works:
ESPHome is a *tool* which aims to make managing your ESP boards as simple as possible. It reads in a YAML configuration file (just like Home Assistant) and creates custom firmware which it installs on your ESP device. Devices or sensors added in ESPHome's configuration will automatically show up in Home Assistant's UI.

Installing ESPHome Dashboard
----------------------------

The ESPHome Dashboard can be installed as a Home Assistant add-on, which you can find in the add-on store in the Supervisor panel. Open it using the following button then click on INSTALL:

.. image:: https://my.home-assistant.io/badges/supervisor_addon.svg

>    :target: https://my.home-assistant.io/redirect/supervisor_addon/?addon=a0d7b954_esphome

Reviewer: Can we open this in a new tab? This currently will take the user away from the guide...

basic "node" by use of the Home Assistant add-on.

But first, here's a very quick introduction to how ESPHome works:

> ESPHome is a *tool* which aims to make managing your ESP boards as simple as possible. It reads in a YAML configuration file (just like Home Assistant) and creates a custom firmware binary which it installs on your ESP device. Devices or sensors added in ESPHome's configuration will automatically show up in Home Assistant's UI.

Reviewer 2: ```suggestion
ESPHome is a *tool* which aims to make managing your ESP boards as simple as possible. It reads in a YAML configuration file (just like Home Assistant) and creates custom firmware which it installs on your ESP device. Devices or sensors added in ESPHome's configuration will automatically show up in Home Assistant's UI.
```

.. note::

    The most difficult part of setting up a new ESPHome device is the initial installation. Installation requires that your ESP device is connected with a cable to a computer. Later updates can be installed wirelessly.

    If you use `Microsoft Edge <https://www.microsoft.com/edge>`_ or `Google Chrome <https://www.google.com/chrome>`_, you will be able to install the initial configuration by connecting your ESP device to the computer that you're using to view the ESPHome Dashboard. If you use another browser, you will have to connect the ESP devices to the machine running the ESPHome Dashboard and Home Assistant.

    If the serial port is not showing up, you might not have the required drivers installed. These drivers work for most ESP devices:

      * CP2102 (square chip): `driver <https://www.silabs.com/products/development-tools/software/usb-to-uart-bridge-vcp-drivers>`__
      * CH341: `driver <https://github.com/nodemcu/nodemcu-devkit/tree/master/Drivers>`__

Dashboard Interface
-------------------

Assuming you created your first configuration file with the wizard, let's take a quick
tour of the ESPHome Dashboard interface.

.. figure:: images/dashboard.png

>     :align: center

Reviewer: Might need to update the screenshots, particularly this one that has `upload`

    :align: center
    :width: 95.0%

The wizard will guide you through creating your first configuration and, depending on your browser, install it on your device. You will need to name your configuration and enter your wireless network settings so that the ESP device can come online and can communicate with Home Assistant.

.. note::

  The most difficult part of a new ESPHome device is the initial installation. Installation requires that your ESP device is connected with a cable to your computer. Once the initial installation is done, future updates can be applied wirelessly.

>   If you use `Microsoft Edge <https://www.microsoft.com/edge>`_ or `Google Chrome <https://www.google.com/chrome>`_, you will be able to install your new configuration via your browser to your ESP device, saving you the hassle of connecting the ESP devices to the machine running Home Assistant.

Reviewer 2: ```suggestion
  If you use `Microsoft Edge <https://www.microsoft.com/edge>`_ or `Google Chrome <https://www.google.com/chrome>`_, you will be able to install the initial configuration via your browser onto your ESP device, saving you the hassle of connecting the ESP devices to the machine running Home Assistant.
```

    :align: center
    :width: 95.0%

The wizard will guide you through creating your first configuration and, depending on your browser, install it on your device. You will need to name your configuration and enter your wireless network settings so that the ESP device can come online and can communicate with Home Assistant.

.. raw:: html

    <a name='webserial'></a>

.. note::

>     The most difficult part of setting a new ESPHome device is the initial installation. Installation requires that your ESP device is connected with a cable to a computer. Later updates can be installed wirelessly.

Reviewer 2: ```suggestion
    The most difficult part of setting up a new ESPHome device is the initial installation. Installation requires that your ESP device is connected with a cable to a computer. Later updates can be installed wirelessly.
```

Connecting your device to Home Assistant
----------------------------------------

Once your configuration is installed on your ESP device and is online, it will be automatically discovered by Home Assistant and offered to set up on your integrations screen:

> .. image:: https://my.home-assistant.io/badges/integrations.svg

Author: ```suggestion
.. image:: https://my.home-assistant.io/badges/config_flow_start.svg
```

>    :target: https://my.home-assistant.io/redirect/integrations/

Reviewer 2: Maybe start a config flow instead?

```suggestion
   :target: https://my.home-assistant.io/redirect/config_flow_start/?domain=esphome
```
